components/infieldbutton/index.css

```diff
@@ -10,27 +10,352 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+			--spectrum-infield-button-background-color-down-quiet,
+			transparent
+		);
+		--mod-infield-button-background-color-key-focus: var(
+			--spectrum-infield-button-background-color-key-focus-quiet,
+			transparent
+		);
+
+		--mod-infield-border-color: var(
+			--spectrum-infield-border-color-quiet,
+			transparent
+		);
+		--mod-infield-button-border-width: var(
+			--spectrum-infield-button-border-width-quiet,
+			0
+		);
+
+		&:disabled {
+			--mod-infield-button-background-color: var(
+				--spectrum-infield-button-background-color-quiet-disabled,
+				transparent
+			);
+			--mod-infield-button-border-color: var(
+				--spectrum-infield-button-border-color-quiet-disabled,
+				transparent
+			);
+		}
+	}
+}
 
 .spectrum-InfieldButton {
-  @inherit: %spectrum-BaseButton;
+	border-style: none;
+	background-color: transparent;
+	cursor: pointer;
 
-  border-style: none;
-  background-color: transparent;
+	display: flex;
+	justify-content: center;
+	align-items: center;
 
-  display: flex;
-  justify-content: center;
-  align-items: center;
+	block-size: var(
+		--mod-infield-button-height,
+		var(--spectrum-infield-button-height)
+	);
+	inline-size: var(
+		--mod-infield-button-width,
+		var(--spectrum-infield-button-width)
+	);
+
+	padding: var(
+		--mod-infield-button-edge-to-fill,
+		var(--spectrum-infield-button-edge-to-fill)
+	);
+
+	.spectrum-InfieldButton-fill {
+		block-size: 100%;
+		inline-size: 100%;
+
+		background-color: var(
+			--mod-infield-button-background-color,
+			var(--spectrum-infield-button-background-color)
+		);
+
+		border-width: var(
+			--mod-infield-button-border-width,
+			var(--spectrum-infield-button-border-width)
+		);
+		border-style: solid;
+		border-color: var(
+			--mod-infield-button-border-color,
+			var(--spectrum-infield-button-border-color)
+		);
+		border-end-end-radius: var(
+			--mod-infield-button-border-radius,
+			var(--spectrum-infield-button-border-radius)
+		);
+		border-start-end-radius: var(
+			--mod-infield-button-border-radius,
+			var(--spectrum-infield-button-border-radius)
+		);
+		border-end-start-radius: var(
+			--mod-infield-button-border-radius,
+			var(--spectrum-infield-button-border-radius)
+		);
+		border-start-start-radius: var(
+			--mod-infield-button-border-radius,
+			var(--spectrum-infield-button-border-radius)
+		);
+
+		padding: var(
+			--mod-infield-button-fill-padding,
+			var(--spectrum-infield-button-fill-padding)
+		);
+	}
+
+	.spectrum-InfieldButton-icon {
+		color: var(
+			--mod-infield-button-icon-color,
+			var(--spectrum-infield-button-icon-color)
+		);
+	}
+
+	&.spectrum-InfieldButton--right {
+		.spectrum-InfieldButton-fill {
+			border-end-start-radius: var(
+				--mod-infield-button-border-radius-reset,
+				var(--spectrum-infield-button-border-radius-reset)
+			);
+			border-start-start-radius: var(
+				--mod-infield-button-border-radius-reset,
+				var(--spectrum-infield-button-border-radius-reset)
+			);
+		}
+	}
+
```

Comment:
Looking at the Figma file it looks like the express theme has all four corners rounded, currently the code is only rounding the right or left corners in express matching the spectrum theme.

Reply:
Oh good catch! Don't know how I missed that